_data/projects/rawsec-inventory.yml

```diff
@@ -14,3 +14,5 @@ tags:
 upforgrabs:
   name: difficulty::easy
   link: https://gitlab.com/rawsec/rawsec-cybersecurity-list/-/issues?label_name%5B%5D=difficulty%3A%3Aeasy
+stats:
+  issue-count: 22
```

Comment:
We don't currently support reading project details from non-GitHub hosts, which is why these details are not populated currently...